python/pyspark/sql/functions.py

```diff
@@ -101,19 +101,19 @@ def lit(col):
 
 
 @since(1.3)
-def col(col):
+def col(col_like):
     """
     Returns a :class:`~pyspark.sql.Column` based on the given column name.'
+    Examples
+    --------
+    >>> df = spark.range(1, 10).toDF('x').withColumn('mod3', col('x') % 3)
+    >>> col('x')
+    >>> column('mod3')
```

Comment:
I think it will fail because the output doesn't match.

```suggestion
    >>> col('x')
    Column<'x'>
    >>> column('mod3')
    Column<'x'>
```